services/workflows-service/src/workflow-defintion/demo-workflow/eu/common/rules.ts

```
@@ -0,0 +1,157 @@
+                  },
+                },
+              },
+            },
+            if: {
+              properties: {
+                country: {
+                  enum: ['AE', 'US', 'CA'],
+                },
+              },
+            },
+            then: {
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)